Fix typo (#3452)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/user-access.md

## Manage via PowerShell

> [!IMPORTANT]

> > New-MsolLicenseOptions will enable all services that were previously disabled unless explictitly identitied in your customized script. As an example, if you wanted to leave both Exchange & Sway disabled while additonally disabling Teams, you'd need to include this in the script or both Exchange & Sway will become enabled for those users you've identified. If you wish to do utilize a GUI to manage this functionality, See: [Office 365 License Reporting and Management Tool -Assign Remove Licenses in Bulk](https://gallery.technet.microsoft.com/Office365-License-cfd9489c)

Reviewer: ```suggestion
> New-MsolLicenseOptions will enable all services that were previously disabled unless explictitly identitied in your customized script. As an example, if you wanted to leave both Exchange & Sway disabled while additonally disabling Teams, you'd need to include this in the script or both Exchange & Sway will become enabled for those users you've identified. To use a GUI to manage this functionality, see [Office 365 License Reporting and Management Tool -Assign Remove Licenses in Bulk](https://gallery.technet.microsoft.com/Office365-License-cfd9489c) for more information.
```
